iMX8Pkg/Include/iMX8.h

```diff
@@ -130,7 +130,7 @@
 #define SDMA2_IRQ               DEFINE_IMX8_INTERRUPT(103)
 #define SDMA3_BASE_ADDRESS      0x302B0000
 #define SDMA3_IRQ               DEFINE_IMX8_INTERRUPT(34)
-#else
+#else // CPU_IMX8MQ
```

Comment:
Looks like SDMA1 and SDMA2 info are the same between IMX8MM and IMX8MQ. Could consider changing the #if to only encompass SDMA3.

Reply:
fixed.